ts/screens/authentication/LandingScreen.tsx

```diff
@@ -296,133 +319,232 @@ export const LandingScreen = () => {
+          // eslint-disable-next-line functional/immutable-data
+          dotTouchCount.current++;
+          if (dotTouchCount.current === 3) {
+            // eslint-disable-next-line functional/immutable-data
+            dotTouchCount.current = 0;
+            navigateToCieUatSelectionScreen();
+          }
+        }}
+      >
+        {carouselCards.map((_, imageIndex) => {
+          const width = scrollX.interpolate({
+            inputRange: [
+              windowWidth * (imageIndex - 1),
+              windowWidth * imageIndex,
+              windowWidth * (imageIndex + 1)
+            ],
+            outputRange: [8, 16, 8],
+            extrapolate: "clamp"
+          });
+          const backgroundColor = scrollX.interpolate({
+            inputRange: [
+              windowWidth * (imageIndex - 1),
+              windowWidth * imageIndex,
+              windowWidth * (imageIndex + 1)
+            ],
+            outputRange: [
+              IOColors["grey-200"],
+              blueColor,
+              IOColors["grey-200"]
+            ],
+            extrapolate: "clamp"
+          });
+          return (
+            <Animated.View
+              key={imageIndex}
+              style={[styles.normalDot, { width, backgroundColor }]}
+            />
+          );
+        })}
+      </View>
+    );
   };
 
-  const handleContinueWithRootOrJailbreak = (continueWith: boolean) => {
-    dispatch(continueWithRootOrJailbreak(continueWith));
+  const Carousel = () => {
+    const cardComponents = renderCardComponents();
+
+    return (
+      <>
+        <ScrollView
+          horizontal={true}
+          pagingEnabled
+          onLayout={() => {
+            // The ScrollView onScrollEndDrag is not called on layout rendering
+            if (!trackFirstCarouselPageOnFirstRenderRef.current) {
+              trackCarousel(0, cardComponents);
+              // eslint-disable-next-line functional/immutable-data
+              trackFirstCarouselPageOnFirstRenderRef.current = true;
+            }
+          }}
+          showsHorizontalScrollIndicator={false}
+          onScrollEndDrag={event => {
+            const contentOffsetX = event.nativeEvent.contentOffset.x;
+            const currentPageIndex = Math.round(contentOffsetX / windowWidth);
+            if (
+              currentPageIndex >= 0 &&
+              cardComponents.length > currentPageIndex
+            ) {
+              trackCarousel(currentPageIndex, cardComponents);
+            }
+          }}
+          onScroll={Animated.event(
+            [
+              {
+                nativeEvent: {
+                  contentOffset: {
+                    x: scrollX
+                  }
+                }
+              }
+            ],
+            { useNativeDriver: false }
+          )}
+          scrollEventThrottle={1}
+        >
+          {cardComponents}
+        </ScrollView>
+        <CarouselDots />
+        <VSpacer size={24} />
```

Comment:
You could use the variable as suggested in comment at line 156 
```suggestion
        <VSpacer size={SPACING_24} />
```

Reply:
As above.